src/Text/Markdown/SlamDown/Html.purs

```diff
@@ -206,14 +224,13 @@ renderHalogen formName (SlamDownState m) (SlamDown bs) = evalState (traverse ren
     modify (+ 1)
     pure (formName ++ "-" ++ show n)
 
-renderTextInput :: forall f. (Alternative f) => String -> String -> TextBoxType -> String -> H.HTML (f SlamDownEvent)
+renderTextInput :: forall f. (Alternative f) => String -> String -> TextBoxType -> Maybe String -> H.HTML (f SlamDownEvent)
 renderTextInput id label t value =
-  H.input [ A.type_ (textBoxTypeName t)
+  H.input ([ A.type_ (textBoxTypeName t)
           , A.id_ id
           , A.name label
-          , A.value value
           , E.onInput (E.input (TextChanged t label))
-          ] []
+          ] <> maybe [] ((\x -> [x]) <<< A.value) value) []
```

Comment:
Extremely minor nitpick, but you could use `singleton` instead of the lambda here I think.